sqlboiler.toml

```diff
@@ -9,4 +9,4 @@ host = "localhost"
 port = 26257
 user = "root"
 sslmode = "disable"
-blacklist = ["goose_db_version"]
+blacklist = ["goose_db_version", "notification_defaults"]
```

Comment:
why is this needed?

Reply:
because it's a view and the upstream sqlboiler crdb driver crashes on encountering one - but I think if we switched over to https://github.com/metal-toolbox/sqlboiler-crdb now it wouldn't be needed (and sqlboiler would do codegen for it)